cfpb/cfgov-refresh
# Spanish home page

Adds the ability to create a Spanish home page based on cf.gov's English home page.

## Additions

- Wagtail page type for `Spanish home page`
- `index_es.html` template, based on `index.html`, with translations

## Testing

1. Check out branch

2. Run `python cfgov/manage.py migrate`

3. Under CFGov, create a new `Spanish home page` and set its language to Spanish. It should look like this:

    ![home-page-es](https://user-images.githubusercontent.com/778171/57819188-2ee4cb00-773c-11e9-8093-8c64c8dc7a9c.png)

4. Test adding featured content using the half width link blob option in the `Header` section of the admin. The content should display in the well below the hero:

    ![home-page-es-featured](https://user-images.githubusercontent.com/778171/57819535-be3eae00-773d-11e9-8911-57177a521ec1.png)
